app/api/v1/auth/index.js

```diff
@@ -175,4 +212,25 @@ router.post('/verification', (req, res, next) => {
   });
 });
 
+/**
+ * Verifies a user's email based on a code sent out to the user's provided email
+ * Responds with a JSON object with fields 'error', 'verified' (if a user is succesfully verified)
+ */
+router.post('/verifyEmail', (req, res, next) => {
+  if (!req.body.code || !req.body.email) {
+    return next(new error.BadRequest('Email and Email verification code must be provided'));
```

Comment:
why require email?

Reply:
Actually not sure, might have been originally me trying to use the email to generate the access code, I'll switch over to the findAcessCode method instead